python/lsst/afw/table/match/matchContinued.py

```diff
@@ -113,13 +114,25 @@ def packMatches(matches):
     - second: the ID of the second source record in each match
     - distance: the distance of each match
 
-    @param[in] matches  Sequence of matches, typically of type SimpleMatch, ReferenceMatch or SourceMatch.
-        Each element must support: `.first.getId()`->int, `.second.getId()->int` and `.distance->float`.
-    @return a catalog of matches.
-
-    @note this pure python implementation exists because SWIG could not easily be used to wrap
-    the overloaded C++ functions, so this was written and tested. It might be practical
-    to wrap the overloaded C++ functions with pybind11, but there didn't seem much point.
+    Parameters
+    ----------
+    matches :
+        Sequence of matches, typically of type SimpleMatch,
+        ReferenceMatch or SourceMatch.  Each element must support:
+        `.first.getId()`->int, `.second.getId()->int` and
+        `.distance->float`.
+
+    Returns
+    -------
+    result :
+        Thecatalog of matches.
+
+    Notes
+    -----
+    This pure python implementation exists because SWIG could not
```

Comment:
Does @TallJimbo want to supply some new text for this note now that we are in the pybind11 era?

Reply:
"The pure python implementation exists as a historical artifact related to SWIG limitations."